src/PerforceCommands.ts

```diff
@@ -152,25 +148,19 @@ export namespace PerforceCommands {
             return false;
         }
 
-        //If folder not opened, overrided p4 directory
         const fileUri = editor.document.uri;
-        const directoryOverride = !checkFolderOpened()
-            ? Path.dirname(fileUri.fsPath)
-            : undefined;
+        p4revert(fileUri);
```

Comment:
If the refresh logic is moved out of the p4Revert function, then it will need to be added here (and in similar places)

Reply:
Hm.  Ok  Ill move it into those functions for now since it's better to be there in one place.

Reply:
moved to p4revert/p4delete calls